extensions/wikia/ThemeDesigner/ThemeSettings.class.php

```diff
@@ -143,14 +143,19 @@ public function getHistory() {
 		}
 
 		foreach ( $history as $key => $val ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$key" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)